src/pygambit/behav.pxi

```diff
@@ -264,7 +264,7 @@ class MixedBehaviorProfile:
         """
         if node.game != self.game:
             raise MismatchError("belief: node must be part of the same game as the profile")
-        return self._belief(node)
+        return self._belief(self.game._resolve_node(node, 'node'))
```

Comment:
The second argument to the `_resolve_*` functions is the name of the function being called (for use in formatting any exceptions that are being raised).  So in this case it would be `'belief'` as the name of the function, instead of `node`.

Reply:
I have addressed this comment.